service/src/main/kotlin/fi/espoo/evaka/shared/dev/DevApi.kt

```diff
@@ -954,11 +953,22 @@ fun Database.Transaction.deleteApplication(id: UUID) {
 }
 
 fun Database.Transaction.deleteAndCascadeEmployee(id: UUID) {
+    execute("DELETE FROM message_account WHERE employee_id = ?", id)
```

Comment:
This will fail if messages are created, but it can be ignored for now and handled when/if this code is ever used in a messaging related test code.